mumbai.hardhat.config.js

```diff
@@ -0,0 +1,90 @@
+/* eslint-disable no-undef */
+const dotenv = require("dotenv");
+
+const dotenvResult = dotenv.config();
+
+if (dotenvResult.error) {
+  throw dotenvResult.error;
+}
+
+require("@nomiclabs/hardhat-ethers");
```

Comment:
I remember discussing and agreeing to go ahead with splitting the hardhat config. But seems like there's too much code duplication to justify having two separate files. We should go back to having a single hardhat config and using env variables to set values.